implementations/rust/ockam/ockam_command/src/node/show.rs

```diff
@@ -72,6 +73,15 @@ fn print_node_info(
 ) {
     println!();
     println!("Node:");
+    if opts.global_args.output_format == OutputFormat::Json {
+        opts.terminal
+            .clone()
+            .stdout()
+            .json(serde_json::json!({"node": { "name": &node_name }}))
```

Comment:
@CAOakleyII after your recent work at https://github.com/build-trust/ockam/pull/5080, I was wondering if we should flatten the structure here to just `{ "name": &node_name }`